packages/teuchos/core/src/Teuchos_ArrayView.hpp

```diff
@@ -315,13 +315,26 @@ T* ArrayView<T>::getRawPtr() const
   return ptr_;
 }
 
+template<class T> inline
+T* ArrayView<T>::data() const
+{
+  debug_assert_valid_ptr();
+  return ptr_;
+}
+
 template<class T> inline
 const T* ArrayView<const T>::getRawPtr() const
 {
   debug_assert_valid_ptr();
   return ptr_;
 }
 
+template<class T> inline
+const T* ArrayView<const T>::data() const
+{
+  debug_assert_valid_ptr();
```

Comment:
Same deal here. To void duplication, why not just call `this->getRawPtr()`?

Reply:
See above.